package.json

```diff
@@ -21,14 +21,14 @@
     "package": "node script/package",
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -p tslint-rules",
-    "lint": "npm run compile:tslint && tslint \"./app/{src,test}/**/*.{ts,tsx}\"",
-    "check-prettiness": "node script/is-it-pretty",
+    "lint": "npm run compile:tslint && npm run tslint && npm run eslint-check && npm run eslint",
+    "tslint": "tslint ./tslint-rules/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}",
+    "eslint": "ts-node script/eslint.ts",
+    "eslint-check": "eslint --print-config .eslintrc.* | eslint-config-prettier-check",
     "publish": "node script/publish",
     "clean-slate": "rimraf out node_modules app/node_modules && npm install",
     "rebuild-hard:dev": "npm run clean-slate && npm run build:dev",
-    "rebuild-hard:prod": "npm run clean-slate && npm run build:prod",
-    "prettier:base": "prettier --single-quote --trailing-comma es5 --no-semi --write",
```

Comment:
@iAmWillShepherd 

> Will removing Prettier also remove our auto-formatting?

Everything is exactly the same, except Prettier is run within ESLint instead of being run separately. Running `npm run eslint -- --fix` will format the code with Prettier along with fixing other errors from other ESLint rules.